src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -201,7 +208,37 @@ public sealed class ProjectInfo
         {
             return Create(
                 id, version, name, assemblyName, language,
-                filePath, outputFilePath, outputRefFilePath: null, compilationOptions, parseOptions,
+                filePath, outputFilePath, outputRefFilePath: null, defaultNamespace: null, compilationOptions, parseOptions,
+                documents, projectReferences, metadataReferences, analyzerReferences, additionalDocuments,
+                isSubmission, hostObjectType, hasAllInformation: true);
+        }
+
+        // 2.8.0 BACKCOMPAT OVERLOAD -- DO NOT TOUCH
+        /// <summary>
+        /// Create a new instance of a ProjectInfo.
+        /// </summary>
```

Comment:
tagging @jasonmalinowski to make sure this is the right way to do things.

Reply:
I reverted the change to this.